std/experimental/allocator/building_blocks/affix_allocator.d

```diff
@@ -520,3 +540,30 @@ struct AffixAllocator(Allocator, Prefix, Suffix = void)
     static assert(is(typeof(a.allocate) == shared));
     assert(buf.length == 10);
 }
+
+@system unittest
```

Comment:
I think it would be a good idea to also have a test in which we access this from multiple threads.